src/views/components/copy-dropdown.html

```diff
@@ -41,5 +41,44 @@
 				canBuildHtml,
 			};
 		},
+		onrender () {
+			let handleDropdownShown = (event) => {
+				let dropdown = $(event.target);
+				dropdown.find('.dropdown-menu').attr('aria-expanded', true);
+
+				dropdown.on('keydown', (e) => {
+					if (e.key === 'ArrowDown' || e.key === 'ArrowUp') {
+						e.preventDefault();
+						let items = dropdown.find('.dropdown-menu li button');
+						let currentIndex = items.index(e.target);
+						let nextIndex = (currentIndex + (e.key === 'ArrowDown' ? 1 : -1)) % items.length;
+
+						if (nextIndex < 0) {
+							nextIndex = items.length - 1;
+						}
+
+						items.eq(nextIndex).focus();
+					}
+				});
+
+				setTimeout(() => {
+					dropdown.find('.dropdown-menu li:first-child button').focus();
+				}, 10);
```

Comment:
This triggers focus and, hence, a visual outline if I open the dropdown with the mouse, which is wrong. But it seems everything works fine without it as well, just requires one more tab?